.circleci/config.yml

```diff
@@ -17,6 +18,9 @@ workflows:
 
 version: 2.1
 
+orbs:
+  shellcheck: circleci/shellcheck@3.1.1
```

Comment:
Yes. I can do it.
Hum...it is not recommended to use the [latest version](https://circleci.com/docs/orbs-faq#how-to-use-the-latest-version-of-an-orb) since it can introduce unexpected breaking changes, but you can set it as `@volatile` to always use the latest if needed. But we can set only the major version as well, so circleCI will always use the latest compatible version up to that major version.

Reply:
I'd just use `volatile` in case of shellcheck specifically since I don't expect much heavy breakage here but in other cases like node.js major version would be fine too.

Breaking changes are not such a big issue for tools we use in CI. We generally prefer to stay on the latest versions and just fix some minor problems they might introduce from time to time. And if the breakage is bigger, we can always hard-code it then. It's a trade-off but I think it's better than using an ancient version and then feeling the pain of upgrade all at once :) With versions hard-coded everywhere it's also hard to keep them up to date manually.